libs/react/src/lib/link/link.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { HTMLProps } from 'react'
+
+interface LinkProps {
+  link?: string
+  text?: string
+}
```

Comment:
probably we should extend the props from default anchor element props, to support accessibility. For instance,
<Link link="some link" text="Read more" aria-label="Read more about this link" />.

Reply:
Alright, do mean to add more of the default anchor element props to the interface?

Reply:
my suggestion is to extend React.HTMLProps<HTMLAnchorElement> but maybe you can omit the children and href, depends on how you want this component to work